gcp-ts-serverless-raw/typescriptfunc/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export const handler = async (req: any, res: any) => {
+  await res.send("Hello World!");
```

Comment:
Little thing, but since the other languages set the response to `text/plain`, it'd be nice to do the same here as well:

```suggestion
  res.set("Content-Type", "text/plain");
  await res.send("Hello World!");
```